code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java

```diff
@@ -54,11 +53,13 @@ void useCredential(AuthorizationRoundTripState state, Credential credential, Htt
     calendarInfo.join();
 
     PositiveUserContacts fullContacts = ofy().load().type(PositiveUserContacts.class).id(state.userId).now();
-    if(fullContacts.getListOfContacts().isEmpty()) {
+    if(fullContacts == null) {
```

Comment:
Do you want to check for both the null case and the empty contact list case?

Reply:
It doesn't matter if it is empty, the set will still add data.

Reply:
Also in the runnable that populates that list, it checks if the list is empty before saving so they should be mutually exclusive.

Reply:
Sounds good to me. Feel free to resolve.

Reply:
meant to say mutually inclusive*